CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 New features, fixed bugs, known defects and other noteworthy changes to each release of the Catena-X Portal helm chart.
 
+## 1.5.1
+
+### Bugfix
+
+* Marketplace Service:  POST: /api/apps/autoSetup fixed mailing for subscription activation
+* Services Service:  POST: /api/services/autoSetup fixed mailing for subscription activation
```

Comment:
Could you please note this as a change? As we did not have a bugfix in the chart.

### Change

* changed to new container images for portal-backend: v1.5.1

Reply:
did change it :)